src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -18,6 +18,7 @@ public void runPolycordBackward(){
 
     public boolean[] ballPositions(){
 
+        return new boolean[1];
```

Comment:
same here

Reply:
There was an issue where the Bool didn't have a return so I put one in as a place holder.